bb/master/config.py

```diff
@@ -128,6 +140,10 @@ class Mode(Enum):
     "centos": {
         "b-1-10": {},
         "b-1-14": {}
+    },
+	 "centos_no_x11": {
+		"b-1-20": {},
+		"b-1-14": {} #temporary solution for tests
```

Comment:
Delete this line immediately after the deploying new `"b-1-20"` worker and testing it.